optimize/util/optimize-commons/src/main/java/io/camunda/optimize/service/util/configuration/db/DatabaseConnection.java

```diff
@@ -26,6 +26,10 @@ public class DatabaseConnection {
   private ProxyConfiguration proxy;
   private Boolean awsEnabled;
 
+  private boolean clusterTaskCheckingEnabled = true;
+
+  private boolean initSchemaEnabled = true;
```

Comment:
I think we should have both of these as `Boolean` and with no default value set. The reasoning being that the `service-config` file remains the source of truth for where the default value gets set

Reply:
fixed